_docs/_latest/setup/upgrade.md

```diff
@@ -5,72 +5,106 @@ order: 00
 
 # {{ page.title }}
 
-This guide explains how to use the Forseti upgrade tool.
+This guide explains how to upgrade your Forseti instance.
 
-## Before you begin
+---
+
+{% capture 1x_upgrade %}
 
-Before you upgrade Forseti, you'll need the following:
+## Important notes
 
- * A G Suite super admin account to complete the [Domain-wide delegation steps]({% link _docs/latest/configure/gsuite.md %}). 
-Forseti 2.0 requires G Suite being enabled, so you'll need this before you start the installation.
+ * We don't support data migration from v1 to v2, so you will need to archive the database manually 
+   for future reference if the data is important to you.
+ * [Forseti v2 configuration]({% link _docs/latest/configure/forseti/index.md %}) is different than v1 so 
+   you can not replace the v2 configuration file with the v1 configuration file.
+ * Configuration of the resources that are inventoried is not configureable.
```

Comment:
I don't understand what this means. Do we mean that "you can't configure resources that are inventoried" or something else?

Reply:
Thanks for helping clarify this offline! How about changing this to:

In v2, all resources are inventoried. You won't be able to configure Inventory to include or exclude resources.

Reply:
Thanks for the suggestion! Updated.